<!DOCTYPE html>
<html lang="en">

<!-- Head tag -->
<head><meta name="generator" content="Hexo 3.8.0">

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!--Description-->

    

    
        <meta name="description" content="PH.D. student">
    

    <!--Author-->
    
        <meta name="author" content="Aoyue Bi">
    

    <!--Open Graph Title-->
    
        <meta property="og:title" content="Plant Genetics">
    

    <!--Open Graph Description-->
    
        <meta property="og:description" content="PH.D. student">
    

    <!--Open Graph Site Name-->
        <meta property="og:site_name" content="Plant Genetics">

    <!--Type page-->
    
        <meta property="og:type" content="website">
    

    <!--Page Cover-->
    
    
        <meta property="og:image" content="http://yoursite.comhttp://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg">
    

        <meta name="twitter:card" content="summary_large_image">

    

    
        <meta name="twitter:image" content="http://yoursite.comhttp://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg">
    

    <!-- Title -->
    
    <title>Plant Genetics</title>

    <!-- Bootstrap Core CSS -->
    <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link rel="stylesheet" href="/css/style.css">

    <!-- Custom Fonts -->
    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" type="text/css">
    <link href="//fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css">
    <link href="//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800" rel="stylesheet" type="text/css">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
    <script src="//oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
    <script src="//oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

    <!-- Gallery -->
    <link href="//cdnjs.cloudflare.com/ajax/libs/featherlight/1.3.5/featherlight.min.css" type="text/css" rel="stylesheet">

    <!-- Google Analytics -->
    


    <!-- favicon -->
    

</head>


<body>

    <!-- Menu -->
    <!-- Navigation -->
<nav class="navbar navbar-default navbar-custom navbar-fixed-top">
    <div class="container-fluid">
        <!-- Brand and toggle get grouped for better mobile display -->
        <div class="navbar-header page-scroll">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">HOME PAGE</a>
        </div>

        <!-- Collect the nav links, forms, and other content for toggling -->
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                
                    <li>
                        <a href="/">
                            
                                Home
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="/archives">
                            
                                Archives
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="/tags">
                            
                                Tags
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="/categories">
                            
                                Categories
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="https://github.com/klugjo/hexo-theme-clean-blog">
                            
                                <i class="fa fa-github fa-stack-2x"></i>
                            
                        </a>
                    </li>
                
            </ul>
        </div>
        <!-- /.navbar-collapse -->
    </div>
    <!-- /.container -->
</nav>

    <!-- Main Content -->
    <!-- Page Header -->
<!-- Set your background image for this header in your post front-matter: cover -->

<header class="intro-header" style="background-image: url('http://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg')">
    <div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <div class="post-heading">
                    <h1>Aoyue's note</h1>
                    
                    <span class="meta">
                        <!-- Date and Author -->
                        
                        
                            2020-08-18
                        
                    </span>
                </div>
            </div>
        </div>
    </div>
</header>

<!-- Post Content -->
<article>
    <div class="container">
        <div class="row">

            <!-- Tags and categories -->
           

            <!-- Gallery -->
            

            <!-- Post Main Content -->
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <p>[toc]</p>
<h1 id="一条染色体的测试，画MAF图"><a href="#一条染色体的测试，画MAF图" class="headerlink" title="一条染色体的测试，画MAF图"></a>一条染色体的测试，画MAF图</h1><p>程序如下：</p>
<pre><code>java -Xmx100g -jar TIGER_v1.0.1.jar -a HapScanner -p ./parameters_036_abd_hapScanner.txt &gt; log_036_abd_hapScanner2.txt 2&gt;&amp;1 &amp;
</code></pre><ul>
<li>chr036 一共合并 call 出 19051 个 Indel, 在 六倍体中 maf 等于0的有 7885 个,小于0.01且不等于0的有 5003 个， 剩余11166 个。</li>
<li>maf 小于 0.01 的有 12888/19051 占比：0.6764<br>结论：14213456个indel中，预计保留 14213456*0.3236 = 4599474</li>
</ul>
<h1 id="路径大全"><a href="#路径大全" class="headerlink" title="路径大全"></a>路径大全</h1><h2 id="HapScanner-VCF-file-文件路径"><a href="#HapScanner-VCF-file-文件路径" class="headerlink" title="HapScanner VCF file 文件路径"></a>HapScanner VCF file 文件路径</h2><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br></pre></td><td class="code"><pre><span class="line">cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d</span><br><span class="line">cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF</span><br><span class="line"></span><br><span class="line">cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd</span><br><span class="line">cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF</span><br><span class="line"></span><br><span class="line">cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab</span><br><span class="line">cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF</span><br></pre></td></tr></table></figure>
<h2 id="bcftools-合并后的VCF文件路径"><a href="#bcftools-合并后的VCF文件路径" class="headerlink" title="bcftools 合并后的VCF文件路径"></a>bcftools 合并后的VCF文件路径</h2><pre><code>cd /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner
</code></pre><h2 id="过滤MAF-MISS-OCCURRENCE-后的VCF文件路"><a href="#过滤MAF-MISS-OCCURRENCE-后的VCF文件路" class="headerlink" title="过滤MAF MISS OCCURRENCE 后的VCF文件路"></a>过滤MAF MISS OCCURRENCE 后的VCF文件路</h2><pre><code>cd /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel
</code></pre><h2 id="log"><a href="#log" class="headerlink" title="log"></a>log</h2><pre><code>cd /data4/home/aoyue/vmap2/analysis/023_hapScanner_basedPopDepth/log/004
</code></pre><h1 id="Pipeline"><a href="#Pipeline" class="headerlink" title="Pipeline"></a>Pipeline</h1><h2 id="Step1-bgzip-and-build-vcf-index"><a href="#Step1-bgzip-and-build-vcf-index" class="headerlink" title="Step1: bgzip and build vcf index"></a>Step1: bgzip and build vcf index</h2><h3 id="abd"><a href="#abd" class="headerlink" title="abd"></a>abd</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br></pre></td><td class="code"><pre><span class="line">bgzip -@ 4 chr001.vcf &amp;&amp; tabix -p vcf chr001.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr002.vcf &amp;&amp; tabix -p vcf chr002.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr003.vcf &amp;&amp; tabix -p vcf chr003.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr004.vcf &amp;&amp; tabix -p vcf chr004.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr005.vcf &amp;&amp; tabix -p vcf chr005.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr006.vcf &amp;&amp; tabix -p vcf chr006.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr007.vcf &amp;&amp; tabix -p vcf chr007.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr008.vcf &amp;&amp; tabix -p vcf chr008.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr009.vcf &amp;&amp; tabix -p vcf chr009.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr010.vcf &amp;&amp; tabix -p vcf chr010.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr011.vcf &amp;&amp; tabix -p vcf chr011.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr012.vcf &amp;&amp; tabix -p vcf chr012.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr013.vcf &amp;&amp; tabix -p vcf chr013.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr014.vcf &amp;&amp; tabix -p vcf chr014.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr015.vcf &amp;&amp; tabix -p vcf chr015.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr016.vcf &amp;&amp; tabix -p vcf chr016.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr017.vcf &amp;&amp; tabix -p vcf chr017.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr018.vcf &amp;&amp; tabix -p vcf chr018.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr019.vcf &amp;&amp; tabix -p vcf chr019.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr020.vcf &amp;&amp; tabix -p vcf chr020.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr021.vcf &amp;&amp; tabix -p vcf chr021.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr022.vcf &amp;&amp; tabix -p vcf chr022.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr023.vcf &amp;&amp; tabix -p vcf chr023.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr024.vcf &amp;&amp; tabix -p vcf chr024.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr025.vcf &amp;&amp; tabix -p vcf chr025.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr026.vcf &amp;&amp; tabix -p vcf chr026.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr027.vcf &amp;&amp; tabix -p vcf chr027.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr028.vcf &amp;&amp; tabix -p vcf chr028.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr029.vcf &amp;&amp; tabix -p vcf chr029.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr030.vcf &amp;&amp; tabix -p vcf chr030.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr031.vcf &amp;&amp; tabix -p vcf chr031.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr032.vcf &amp;&amp; tabix -p vcf chr032.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr033.vcf &amp;&amp; tabix -p vcf chr033.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr034.vcf &amp;&amp; tabix -p vcf chr034.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr035.vcf &amp;&amp; tabix -p vcf chr035.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr036.vcf &amp;&amp; tabix -p vcf chr036.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr037.vcf &amp;&amp; tabix -p vcf chr037.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr038.vcf &amp;&amp; tabix -p vcf chr038.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr039.vcf &amp;&amp; tabix -p vcf chr039.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr040.vcf &amp;&amp; tabix -p vcf chr040.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr041.vcf &amp;&amp; tabix -p vcf chr041.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr042.vcf &amp;&amp; tabix -p vcf chr042.vcf.gz &amp;</span><br></pre></td></tr></table></figure>
<pre><code>cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF
</code></pre><h3 id="ab"><a href="#ab" class="headerlink" title="ab"></a>ab</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br></pre></td><td class="code"><pre><span class="line">bgzip -@ 4 chr001.vcf &amp;&amp; tabix -p vcf chr001.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr002.vcf &amp;&amp; tabix -p vcf chr002.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr003.vcf &amp;&amp; tabix -p vcf chr003.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr004.vcf &amp;&amp; tabix -p vcf chr004.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr007.vcf &amp;&amp; tabix -p vcf chr007.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr008.vcf &amp;&amp; tabix -p vcf chr008.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr009.vcf &amp;&amp; tabix -p vcf chr009.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr010.vcf &amp;&amp; tabix -p vcf chr010.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr013.vcf &amp;&amp; tabix -p vcf chr013.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr014.vcf &amp;&amp; tabix -p vcf chr014.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr015.vcf &amp;&amp; tabix -p vcf chr015.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr016.vcf &amp;&amp; tabix -p vcf chr016.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr019.vcf &amp;&amp; tabix -p vcf chr019.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr020.vcf &amp;&amp; tabix -p vcf chr020.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr021.vcf &amp;&amp; tabix -p vcf chr021.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr022.vcf &amp;&amp; tabix -p vcf chr022.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr025.vcf &amp;&amp; tabix -p vcf chr025.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr026.vcf &amp;&amp; tabix -p vcf chr026.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr027.vcf &amp;&amp; tabix -p vcf chr027.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr028.vcf &amp;&amp; tabix -p vcf chr028.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr031.vcf &amp;&amp; tabix -p vcf chr031.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr032.vcf &amp;&amp; tabix -p vcf chr032.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr033.vcf &amp;&amp; tabix -p vcf chr033.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr034.vcf &amp;&amp; tabix -p vcf chr034.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr037.vcf &amp;&amp; tabix -p vcf chr037.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr038.vcf &amp;&amp; tabix -p vcf chr038.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr039.vcf &amp;&amp; tabix -p vcf chr039.vcf.gz &amp;</span><br><span class="line">bgzip -@ 4 chr040.vcf &amp;&amp; tabix -p vcf chr040.vcf.gz &amp;</span><br></pre></td></tr></table></figure>
<pre><code>cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF
</code></pre><h2 id="Step2-bcftools-merge"><a href="#Step2-bcftools-merge" class="headerlink" title="Step2: bcftools merge"></a>Step2: bcftools merge</h2><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br></pre></td><td class="code"><pre><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr001.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr001.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr001.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr002.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr002.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr002.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr003.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr003.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr003.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr004.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr004.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr004.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr005.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr005.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr005.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr006.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr006.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr006.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr007.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr007.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr007.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr008.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr008.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr008.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr009.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr009.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr009.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr010.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr010.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr010.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr011.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr011.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr011.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr012.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr012.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr012.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr013.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr013.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr013.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr014.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr014.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr014.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr015.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr015.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr015.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr016.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr016.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr016.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr017.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr017.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr017.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr018.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr018.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr018.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr019.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr019.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr019.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr020.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr020.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr020.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr021.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr021.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr021.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr022.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr022.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr022.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr023.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr023.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr023.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr024.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr024.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr024.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr025.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr025.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr025.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr026.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr026.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr026.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr027.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr027.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr027.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr028.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr028.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr028.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr029.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr029.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr029.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr030.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr030.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr030.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr031.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr031.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr031.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr032.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr032.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr032.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr033.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr033.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr033.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr034.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr034.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr034.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr035.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr035.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr035.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr036.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr036.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr036.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr037.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr037.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr037.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr038.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr038.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr038.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr039.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr039.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr039.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr040.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/ab/out/VCF/chr040.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr040.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr041.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr041.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr041.vcf &amp;</span><br><span class="line">/data1/programs/bcftools-1.8/bcftools merge -m all --force-samples -f PASS,. --threads 8 /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/abd/out/VCF/chr042.vcf.gz /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/d/out/VCF/chr042.vcf.gz -o /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr042.vcf &amp;</span><br></pre></td></tr></table></figure>
<pre><code>cd /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner
</code></pre><h2 id="Step3-filter-by-maf-gt-0-01-Occurrence-gt-2-Miss-lt-0-2"><a href="#Step3-filter-by-maf-gt-0-01-Occurrence-gt-2-Miss-lt-0-2" class="headerlink" title="Step3: filter by maf&gt;=0.01, Occurrence&gt;=2, Miss&lt;=0.2"></a>Step3: filter by maf&gt;=0.01, Occurrence&gt;=2, Miss&lt;=0.2</h2><h3 id="结果文件"><a href="#结果文件" class="headerlink" title="结果文件"></a>结果文件</h3><pre><code>cd /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel
</code></pre><h3 id="日志文件目录"><a href="#日志文件目录" class="headerlink" title="日志文件目录"></a>日志文件目录</h3><pre><code>cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence
</code></pre><h3 id="运行程序"><a href="#运行程序" class="headerlink" title="运行程序"></a>运行程序</h3><p>Note: -Xms -Xmx 根据服务器大小进行设置，在数据量非常小的情况下，也可以不设置。<br><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br></pre></td><td class="code"><pre><span class="line">java -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr001.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr001.vcf 2 &gt;&amp;1 &amp;</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr002.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr002.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr003.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr003.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr004.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr004.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr005.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr005.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr006.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr006.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr007.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr007.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr008.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr008.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr009.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr009.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr010.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr010.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr011.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr011.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr012.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr012.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr013.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr013.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr014.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr014.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr015.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr015.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr016.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr016.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr017.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr017.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr018.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr018.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr019.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr019.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr020.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr020.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr021.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr021.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr022.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr022.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr023.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr023.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr024.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr024.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr025.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr025.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr026.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr026.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr027.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr027.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr028.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr028.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr029.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr029.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr030.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr030.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr031.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr031.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr032.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr032.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr033.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr033.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr034.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr034.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr035.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr035.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr036.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr036.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr037.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr037.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr038.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr038.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr039.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr039.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr040.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr040.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr041.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr041.vcf</span><br><span class="line">java -Xms200g -Xmx200g -jar 044_filterMafOccurrenceMiss.jar /data4/home/aoyue/vmap2/genotype/mergedVCF/106_MergedIndel_fromHapScanner/chr042.vcf /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence/log_chr042.vcf</span><br></pre></td></tr></table></figure></p>
<pre><code>脚本运行
nohup sh sh_java.sh &gt; log_java_20200819.txt 2&gt;&amp;1 &amp;
nohup sh sh_java2.sh &gt; log_filtermaf_Indel.txt 2&gt;&amp;1 &amp;
</code></pre><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">cd /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel</span><br><span class="line">cd /data4/home/aoyue/vmap2/analysis/029_hapScanner_onlyIndel/log_filterMafMissOccurrence</span><br><span class="line">ls|xargs cat|grep complete</span><br></pre></td></tr></table></figure>
<h2 id="Step4-修改名字"><a href="#Step4-修改名字" class="headerlink" title="Step4: 修改名字"></a>Step4: 修改名字</h2><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br></pre></td><td class="code"><pre><span class="line">mv chr001_occu2_maf0.01_miss0.2.vcf chr001_vmap2.0.vcf</span><br><span class="line">mv chr002_occu2_maf0.01_miss0.2.vcf chr002_vmap2.0.vcf</span><br><span class="line">mv chr003_occu2_maf0.01_miss0.2.vcf chr003_vmap2.0.vcf</span><br><span class="line">mv chr004_occu2_maf0.01_miss0.2.vcf chr004_vmap2.0.vcf</span><br><span class="line">mv chr005_occu2_maf0.01_miss0.2.vcf chr005_vmap2.0.vcf</span><br><span class="line">mv chr006_occu2_maf0.01_miss0.2.vcf chr006_vmap2.0.vcf</span><br><span class="line">mv chr007_occu2_maf0.01_miss0.2.vcf chr007_vmap2.0.vcf</span><br><span class="line">mv chr008_occu2_maf0.01_miss0.2.vcf chr008_vmap2.0.vcf</span><br><span class="line">mv chr009_occu2_maf0.01_miss0.2.vcf chr009_vmap2.0.vcf</span><br><span class="line">mv chr010_occu2_maf0.01_miss0.2.vcf chr010_vmap2.0.vcf</span><br><span class="line">mv chr011_occu2_maf0.01_miss0.2.vcf chr011_vmap2.0.vcf</span><br><span class="line">mv chr012_occu2_maf0.01_miss0.2.vcf chr012_vmap2.0.vcf</span><br><span class="line">mv chr013_occu2_maf0.01_miss0.2.vcf chr013_vmap2.0.vcf</span><br><span class="line">mv chr014_occu2_maf0.01_miss0.2.vcf chr014_vmap2.0.vcf</span><br><span class="line">mv chr015_occu2_maf0.01_miss0.2.vcf chr015_vmap2.0.vcf</span><br><span class="line">mv chr016_occu2_maf0.01_miss0.2.vcf chr016_vmap2.0.vcf</span><br><span class="line">mv chr017_occu2_maf0.01_miss0.2.vcf chr017_vmap2.0.vcf</span><br><span class="line">mv chr018_occu2_maf0.01_miss0.2.vcf chr018_vmap2.0.vcf</span><br><span class="line">mv chr019_occu2_maf0.01_miss0.2.vcf chr019_vmap2.0.vcf</span><br><span class="line">mv chr020_occu2_maf0.01_miss0.2.vcf chr020_vmap2.0.vcf</span><br><span class="line">mv chr021_occu2_maf0.01_miss0.2.vcf chr021_vmap2.0.vcf</span><br><span class="line">mv chr022_occu2_maf0.01_miss0.2.vcf chr022_vmap2.0.vcf</span><br><span class="line">mv chr023_occu2_maf0.01_miss0.2.vcf chr023_vmap2.0.vcf</span><br><span class="line">mv chr024_occu2_maf0.01_miss0.2.vcf chr024_vmap2.0.vcf</span><br><span class="line">mv chr025_occu2_maf0.01_miss0.2.vcf chr025_vmap2.0.vcf</span><br><span class="line">mv chr026_occu2_maf0.01_miss0.2.vcf chr026_vmap2.0.vcf</span><br><span class="line">mv chr027_occu2_maf0.01_miss0.2.vcf chr027_vmap2.0.vcf</span><br><span class="line">mv chr028_occu2_maf0.01_miss0.2.vcf chr028_vmap2.0.vcf</span><br><span class="line">mv chr029_occu2_maf0.01_miss0.2.vcf chr029_vmap2.0.vcf</span><br><span class="line">mv chr030_occu2_maf0.01_miss0.2.vcf chr030_vmap2.0.vcf</span><br><span class="line">mv chr031_occu2_maf0.01_miss0.2.vcf chr031_vmap2.0.vcf</span><br><span class="line">mv chr032_occu2_maf0.01_miss0.2.vcf chr032_vmap2.0.vcf</span><br><span class="line">mv chr033_occu2_maf0.01_miss0.2.vcf chr033_vmap2.0.vcf</span><br><span class="line">mv chr034_occu2_maf0.01_miss0.2.vcf chr034_vmap2.0.vcf</span><br><span class="line">mv chr035_occu2_maf0.01_miss0.2.vcf chr035_vmap2.0.vcf</span><br><span class="line">mv chr036_occu2_maf0.01_miss0.2.vcf chr036_vmap2.0.vcf</span><br><span class="line">mv chr037_occu2_maf0.01_miss0.2.vcf chr037_vmap2.0.vcf</span><br><span class="line">mv chr038_occu2_maf0.01_miss0.2.vcf chr038_vmap2.0.vcf</span><br><span class="line">mv chr039_occu2_maf0.01_miss0.2.vcf chr039_vmap2.0.vcf</span><br><span class="line">mv chr040_occu2_maf0.01_miss0.2.vcf chr040_vmap2.0.vcf</span><br><span class="line">mv chr041_occu2_maf0.01_miss0.2.vcf chr041_vmap2.0.vcf</span><br><span class="line">mv chr042_occu2_maf0.01_miss0.2.vcf chr042_vmap2.0.vcf</span><br></pre></td></tr></table></figure>
<h2 id="Step5-calculate-Indel-and-make-table"><a href="#Step5-calculate-Indel-and-make-table" class="headerlink" title="Step5: calculate Indel and make table"></a>Step5: calculate Indel and make table</h2><h3 id="在HPC上计数"><a href="#在HPC上计数" class="headerlink" title="在HPC上计数"></a>在HPC上计数</h3><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">cda</span><br><span class="line">java -jar 043_countSitesinFastCallformat.jar  /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel &gt; log_043_countSitesinFastCallformat_fixVMap2.0Indels_20200818.txt 2&gt;&amp;1 &amp;</span><br><span class="line">cat /data4/home/aoyue/vmap2/aaPlantGenetics/log_043_countSitesinFastCallformat_fixVMap2.0Indels_20200818.txt</span><br></pre></td></tr></table></figure>
<h3 id="转化表格"><a href="#转化表格" class="headerlink" title="转化表格"></a>转化表格</h3><p>①将数据放入特定目录下，运行程序<br><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br></pre></td><td class="code"><pre><span class="line">CountSites.mergeChr1and2txt_int(&quot;/Users/Aoyue/project/wheatVMapII/003_dataAnalysis/005_vcf/029_countSiteSummary/002_vmap2.0/log_043_countSitesinFastCallformat_fixVMap2.0Indels_20200818.txt&quot;,&quot;/Users/Aoyue/project/wheatVMapII/003_dataAnalysis/005_vcf/029_countSiteSummary/002_vmap2.0/CountVariants_fixVMap2.0_Indel_20200818.txt&quot;);</span><br></pre></td></tr></table></figure></p>
<h2 id="Step6-压缩并建立索引"><a href="#Step6-压缩并建立索引" class="headerlink" title="Step6: 压缩并建立索引"></a>Step6: 压缩并建立索引</h2><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br><span class="line">4</span><br><span class="line">5</span><br><span class="line">6</span><br><span class="line">7</span><br><span class="line">8</span><br><span class="line">9</span><br><span class="line">10</span><br><span class="line">11</span><br><span class="line">12</span><br><span class="line">13</span><br><span class="line">14</span><br><span class="line">15</span><br><span class="line">16</span><br><span class="line">17</span><br><span class="line">18</span><br><span class="line">19</span><br><span class="line">20</span><br><span class="line">21</span><br><span class="line">22</span><br><span class="line">23</span><br><span class="line">24</span><br><span class="line">25</span><br><span class="line">26</span><br><span class="line">27</span><br><span class="line">28</span><br><span class="line">29</span><br><span class="line">30</span><br><span class="line">31</span><br><span class="line">32</span><br><span class="line">33</span><br><span class="line">34</span><br><span class="line">35</span><br><span class="line">36</span><br><span class="line">37</span><br><span class="line">38</span><br><span class="line">39</span><br><span class="line">40</span><br><span class="line">41</span><br><span class="line">42</span><br><span class="line">43</span><br></pre></td><td class="code"><pre><span class="line">cd /data4/home/aoyue/vmap2/genotype/mergedVCF/107_VMap2.0_Indel</span><br><span class="line">bgzip chr001_vmap2.0.vcf &amp;&amp; tabix -p vcf chr001_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr002_vmap2.0.vcf &amp;&amp; tabix -p vcf chr002_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr003_vmap2.0.vcf &amp;&amp; tabix -p vcf chr003_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr004_vmap2.0.vcf &amp;&amp; tabix -p vcf chr004_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr005_vmap2.0.vcf &amp;&amp; tabix -p vcf chr005_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr006_vmap2.0.vcf &amp;&amp; tabix -p vcf chr006_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr007_vmap2.0.vcf &amp;&amp; tabix -p vcf chr007_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr008_vmap2.0.vcf &amp;&amp; tabix -p vcf chr008_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr009_vmap2.0.vcf &amp;&amp; tabix -p vcf chr009_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr010_vmap2.0.vcf &amp;&amp; tabix -p vcf chr010_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr011_vmap2.0.vcf &amp;&amp; tabix -p vcf chr011_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr012_vmap2.0.vcf &amp;&amp; tabix -p vcf chr012_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr013_vmap2.0.vcf &amp;&amp; tabix -p vcf chr013_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr014_vmap2.0.vcf &amp;&amp; tabix -p vcf chr014_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr015_vmap2.0.vcf &amp;&amp; tabix -p vcf chr015_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr016_vmap2.0.vcf &amp;&amp; tabix -p vcf chr016_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr017_vmap2.0.vcf &amp;&amp; tabix -p vcf chr017_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr018_vmap2.0.vcf &amp;&amp; tabix -p vcf chr018_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr019_vmap2.0.vcf &amp;&amp; tabix -p vcf chr019_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr020_vmap2.0.vcf &amp;&amp; tabix -p vcf chr020_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr021_vmap2.0.vcf &amp;&amp; tabix -p vcf chr021_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr022_vmap2.0.vcf &amp;&amp; tabix -p vcf chr022_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr023_vmap2.0.vcf &amp;&amp; tabix -p vcf chr023_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr024_vmap2.0.vcf &amp;&amp; tabix -p vcf chr024_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr025_vmap2.0.vcf &amp;&amp; tabix -p vcf chr025_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr026_vmap2.0.vcf &amp;&amp; tabix -p vcf chr026_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr027_vmap2.0.vcf &amp;&amp; tabix -p vcf chr027_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr028_vmap2.0.vcf &amp;&amp; tabix -p vcf chr028_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr029_vmap2.0.vcf &amp;&amp; tabix -p vcf chr029_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr030_vmap2.0.vcf &amp;&amp; tabix -p vcf chr030_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr031_vmap2.0.vcf &amp;&amp; tabix -p vcf chr031_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr032_vmap2.0.vcf &amp;&amp; tabix -p vcf chr032_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr033_vmap2.0.vcf &amp;&amp; tabix -p vcf chr033_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr034_vmap2.0.vcf &amp;&amp; tabix -p vcf chr034_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr035_vmap2.0.vcf &amp;&amp; tabix -p vcf chr035_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr036_vmap2.0.vcf &amp;&amp; tabix -p vcf chr036_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr037_vmap2.0.vcf &amp;&amp; tabix -p vcf chr037_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr038_vmap2.0.vcf &amp;&amp; tabix -p vcf chr038_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr039_vmap2.0.vcf &amp;&amp; tabix -p vcf chr039_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr040_vmap2.0.vcf &amp;&amp; tabix -p vcf chr040_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr041_vmap2.0.vcf &amp;&amp; tabix -p vcf chr041_vmap2.0.vcf.gz &amp;</span><br><span class="line">bgzip chr042_vmap2.0.vcf &amp;&amp; tabix -p vcf chr042_vmap2.0.vcf.gz &amp;</span><br></pre></td></tr></table></figure>
<h2 id="Step7-和VMap2-0进行合并-压缩并建立索引"><a href="#Step7-和VMap2-0进行合并-压缩并建立索引" class="headerlink" title="Step7: 和VMap2.0进行合并, 压缩并建立索引"></a>Step7: 和VMap2.0进行合并, 压缩并建立索引</h2><ul>
<li>将2020-05 VMap2.0 (i.e. hapscanner并过滤 MAF&gt;=0.01 Miss&lt;=0.2 Occurrence &gt;=2) 中的 SNP 和Indel 结果只保留SNP(include biallelic and tri-allelic SNP)</li>
<li>将2020-08 新做出来Indel (revised hapscanner output –&gt; 并过滤 MAF&gt;=0.01 Miss&lt;=0.2 Occurrence &gt;=2) 和上文SNP排序并合并</li>
<li>见 java finalizeVMapII 方法</li>
</ul>
<h2 id="Step8-最终核查所有染色体位点数是否一致-很重要的一步。"><a href="#Step8-最终核查所有染色体位点数是否一致-很重要的一步。" class="headerlink" title="Step8: 最终核查所有染色体位点数是否一致,很重要的一步。"></a>Step8: 最终核查所有染色体位点数是否一致,很重要的一步。</h2><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br><span class="line">3</span><br></pre></td><td class="code"><pre><span class="line">cda</span><br><span class="line">java -jar 043_countSitesinFastCallformat.jar  /data4/home/aoyue/vmap2/genotype/mergedVCF/108_VMap2.0_final &gt; log_043_countSitesinFastCallformat_fixVMap2.0_20200823.txt 2&gt;&amp;1 &amp;</span><br><span class="line">cat /data4/home/aoyue/vmap2/aaPlantGenetics/log_043_countSitesinFastCallformat_fixVMap2.0_20200823.txt</span><br></pre></td></tr></table></figure>
<h2 id="Step9-建立md5，确定最终版-并拷贝至移动硬盘保存"><a href="#Step9-建立md5，确定最终版-并拷贝至移动硬盘保存" class="headerlink" title="Step9: 建立md5，确定最终版,并拷贝至移动硬盘保存"></a>Step9: 建立md5，确定最终版,并拷贝至移动硬盘保存</h2><figure class="highlight plain"><table><tr><td class="gutter"><pre><span class="line">1</span><br><span class="line">2</span><br></pre></td><td class="code"><pre><span class="line">ls *.gz|xargs -I file echo &quot;md5sum file &gt; file.md5&quot; &gt; md5.sh</span><br><span class="line">cp -Rf /data4/home/aoyue/vmap2/genotype/mergedVCF/108_VMap2.0_final/* /mnt/usb/WheatVariationMapII/VMapII/VMap2.0 &amp;</span><br></pre></td></tr></table></figure>


                
            </div>

            <!-- Comments -->
            
                <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                    


                </div>
            
        </div>
    </div>
</article>

    <!-- Footer -->
    <hr>

<!-- Footer -->
<footer>
    <div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <ul class="list-inline text-center">
                    

                    

                    
                        <li>
                            <a href="https://github.com/klugjo/hexo-theme-clean-blog" target="_blank">
                                <span class="fa-stack fa-lg">
                                    <i class="fa fa-circle fa-stack-2x"></i>
                                    <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                                </span>
                            </a>
                        </li>
                    

                    

                    

                    
                </ul>
                <p class="copyright text-muted">&copy; 2020 Aoyue Bi<br></p>
                <p class="copyright text-muted">Original Theme <a target="_blank" href="http://startbootstrap.com/template-overviews/clean-blog/">Clean Blog</a> from <a href="http://startbootstrap.com/" target="_blank">Start Bootstrap</a></p>
                <p class="copyright text-muted">Adapted for <a target="_blank" href="https://hexo.io/">Hexo</a> by <a href="http://www.codeblocq.com/" target="_blank">Jonathan Klughertz</a></p>
            </div>
        </div>
    </div>
</footer>


    <!-- After footer scripts -->
    
<!-- jQuery -->
<script src="//code.jquery.com/jquery-2.1.4.min.js"></script>

<!-- Bootstrap -->
<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>

<!-- Gallery -->
<script src="//cdnjs.cloudflare.com/ajax/libs/featherlight/1.3.5/featherlight.min.js" type="text/javascript" charset="utf-8"></script>

<!-- Disqus Comments -->



</body>

</html>